pointcloud_topic:
    camera_topic: /camera/depth/color/points

reference_frame:
    frame_id: camera_depth_optical_frame

voxel_grid:
    x: 0.02
    y: 0.02
    z: 0.02
    enable: false
    
pass_through:
    z_min: 0.0
    z_max: 2.0
    enable: false

segmentation:
    distance_threshold: 0.01
    enable: false
    
outlier_removal:
    meanK: 50
    standard_dev_mult: 1.0
    enable: false
    
cluster_extraction:
    cluster_tolerance: 0.06
    min_cluster_size: 10
    max_cluster_size: 250
    enable: false
    
    
    

